time.c

```diff
@@ -63,8 +63,17 @@ uint64_t usec_sleep(struct thread_data *td, unsigned long usec)
 		req.tv_nsec = ts * 1000;
 		fio_gettime(&tv, NULL);
 
-		if (nanosleep(&req, NULL) < 0)
+		if (req.tv_sec) {
+			if (sleep(req.tv_sec) != 0)
 			break;
+		}
+		if (ts) {
+			if (usleep(ts) < 0)
+			break;
+		}
+		// nanosleep was hanging on Windows 7 so replaced
+		//if (nanosleep(&req, NULL) < 0)
+		//	break;
```

Comment:
Rather than just stubbing this out we need to know why it went wrong. @phoulihan9 do you have the workload that generated the problem? Perhaps @bcran has some suggestions on why this happened?